cli/tests/extensions.rs

```diff
@@ -261,6 +261,36 @@ fn conflicting_extension_name_is_filtered() {
     assert!(output.contains("extension was filtered out"));
 }
 
+#[test]
+fn api_extension() {
+    let tmp_dir = TmpDir::new();
+    Command::cargo_bin("phylum")
+        .unwrap()
+        .env("XDG_DATA_HOME", &tmp_dir)
+        .arg("extension")
+        .arg("add")
+        .arg(fixtures_path().join("api-extension"))
+        .assert()
+        .success();
+
+    let cmd = Command::cargo_bin("phylum")
+        .unwrap()
+        .env("XDG_DATA_HOME", &tmp_dir)
+        .arg("api-extension")
+        .assert()
+        .success();
+
+    {
+        let output = std::str::from_utf8(&cmd.get_output().stdout).unwrap();
+        println!("{}", output);
+    }
+    {
+        let output = std::str::from_utf8(&cmd.get_output().stderr).unwrap();
+        println!("{}", output);
+    }
```

Comment:
Yeah, indeed -- the test is poorly set up since at this point I'd only be checking that nothing blows up, really. We need a mocking setup for these tests anyway, which I think we should address as part of a dedicated ticket.

Reply:
I mean other tests already do the output check. I think that doesn't require waiting.

If you're holding off on fleshing out this test because you cannot implement it yet, then I wouldn't add the test at all rather than adding a useless test that doesn't do what it's supposed to yet.

Reply:
Indeed, it is more of a placeholder atm, though I did use the output for manually testing some concepts. We can remove it.